python/paddle/fluid/dygraph/io.py

```diff
@@ -977,3 +988,219 @@ def _output_spec(self, method_name='forward'):
             output_spec.append(spec)
 
         return output_spec
+
+
+def _run_static_graph(input, program_holder, trace_program):
```

Comment:
It is better to move the utility function above the calling function

Reply:
Done,thx.